Gemfile.lock

```diff
@@ -253,4 +253,4 @@ DEPENDENCIES
   webmock
 
 BUNDLED WITH
-   2.0.1
+   1.16.1
```

Comment:
`gem install bundler && bundle` to get rid of this change.

Reply:
Ran the gem install bundler && bundle, and it updated to 2.0.2 (which I assume is fine?)